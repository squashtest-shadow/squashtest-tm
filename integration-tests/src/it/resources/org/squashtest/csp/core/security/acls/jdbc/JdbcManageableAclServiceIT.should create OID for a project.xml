<?xml version='1.0' encoding='UTF-8'?>
<!DOCTYPE dataset SYSTEM "../../../../../../../dtd/squash-tm.1.9.10.dtd">

<dataset>
	<ACL_CLASS ID="-100" CLASSNAME="foo.Bar" />	
	<acl_object_identity />
	<acl_group_permission />
	<acl_responsibility_scope_entry />
</dataset>