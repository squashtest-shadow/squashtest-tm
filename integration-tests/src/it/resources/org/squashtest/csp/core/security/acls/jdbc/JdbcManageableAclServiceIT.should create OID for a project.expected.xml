<?xml version='1.0' encoding='UTF-8'?>
<!DOCTYPE dataset SYSTEM "../../../../../../../dtd/squash-tm.1.9.10.dtd">

<dataset>
	<ACL_OBJECT_IDENTITY CLASS_ID="100" IDENTITY="10" />
</dataset>